### DISCUSSION TITLE
2022.09.17 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#84] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 민코딩 진도 따라가기, velog 글 7개 작성, 자바의 정석 요약본 1회독 하기

점수 : 30점
이유 : 
- 민코딩 진도 따라가기 - failed
민코딩은 아직 못 푼 문제가 많다... 내일까진 다 푸는게 목표..
- velog 글 7개 작성 - succeeded
벨로그 글은 급하게 작성을 마쳤다. 완성도는 어떠할진 모르겠으나, 일단 갯수는 채웠으니 성공
- 자바의 정석 요약본 1회독 하기 - failed
자바의 정석 요약본은 아직 펴보지도 못했다. 확실히 알고리즘 주간이라 시간이 부족하다....(는 핑계임)

**Comment by giokim12** [comment]
이번주 목표 : 이번주 수업 진도 따라가기, 기업 지원(3개)
점수: 30점
이번주 수업 진도 따라가기: 목금 수업 X... 교수님은 날 버렸다...
기업 지원 (3개): 2개썼음

**Comment by moonthree** [comment]
이번주 목표 : 하루에 인강 30분이상 듣기(중요), 민코딩 따라가기, 복싱 2회 출석, 블로그 글 7개 작성
점수 : 25점
이유 :
- 인강 중요한데 하나도 못 들었다.
- 민코딩.. dfs에게 당해서 못 따라갔다.
- 복싱 1회 출석
- 블로그 2개 작성

총평 : 알고리즘에게 잡혔다..

**Comment by HyeseungNA** [comment]
목표:1일 1알고리즘, 수업 복습하기, 못 따라갔던 민코딩 진도-1일 최소 한문제 풀기
점수 : 10점
이유:
알고리즘 4문제 풀었습니다. 
수업은 진도 이해 절반 정도 했습니다...
민코딩은 한 문제도 풀지 못했습니다. ..

**Comment by YoungSeok222222** [comment]
이번주 목표: 커리큘럼 잘 따라가기, 기업 채용 보고 지원하기
점수: 30점
이유: 커리큘럼에 따라 DFS 문제를 풀고 있지만, 42.5를 풀기 힘들 것 같고, 기업 채용은 너무 적게 지원한것 같습니다..

**Comment by swjiae** [comment]
이번주 목표 : 민코딩 진도 따라가기, SWEA 과제 끝내기, 알고리즘 배웠던 것 복습하기, 주 3회 운동
점수 : 20점
이유
민코딩 진도 못따라감
swea stack만 품
과거 배웠던 알고리즘 복습 못함
주 3회 홈트 or 걷기

**Comment by sooyeonlee127** [comment]
이번주 목표 : 정처기 필기 인강 1강 이상 듣기, 블로그 글 7개 이상 작성, 과제하기
점수: 50점
이유:  
정처기 인강 또 못들었다
블로그 글은 9개 작성했다
과제도 아직 다 못한 상태여서.. 다음주에는 정말 목표 달성할 수 있도록 노력하겠습니다..!
